### PR TITLE
Update Distributed Tracing Page

### PR DESCRIPTION
**Pull request by lizokm**
Make a few more tweaks to make this page easier to digest.

### CODE REVIEW

File: docs/concepts/key-terms/tracing/distributed-tracing.mdx [hunk 1]
```diff
@@ -21,7 +21,45 @@ It's important to note that tracing is not [profiling](https://github.com/produc
...
+
+#### Traces
+
+Traces are defined as the collection of all transactions that share a `trace_id` value.
+
+#### Transactions
+
+Transactions share most properties (start and end time, tags, etc.) with their root spans. They also have a `transaction_name` property, used in the UI to identify the transaction. Common examples include endpoint paths for backend request transactions, task names for cron job transactions, and URLs for page-load transactions.
+
+Before the transaction is sent, the `tags` and `data` properties will get merged with data from the global scope. (Global scope data is set in `Sentry.init()` or by using `Sentry.configureScope()`, `Sentry.setTag()`, `Sentry.setUser()`, and `Sentry.setExtra()`.)
```
Comment (coolguyzone) [inline docs/concepts/key-terms/tracing/distributed-tracing.mdx:36]: Should this be an 'or'? It reads a bit like I would need to use all those functions combined to set the data.

File: docs/concepts/key-terms/tracing/distributed-tracing.mdx [hunk 2]
```diff
@@ -21,7 +21,45 @@ It's important to note that tracing is not [profiling](https://github.com/produc
...
+
+Transaction names can contain **sensitive data**. See [Scrubbing Sensitive Data](/platforms/javascript/data-management/sensitive-data/#scrubbing-data) for more information.
```
Comment (coolguyzone) [inline docs/concepts/key-terms/tracing/distributed-tracing.mdx:38]: This content appears to be duplicated in the note below, let's remove it here.

**Review by coolguyzone — APPROVED** [review]
Added a couple comments, overall looks good! 👍

File: docs/concepts/key-terms/tracing/distributed-tracing.mdx [hunk 3]
```diff
@@ -21,7 +21,45 @@ It's important to note that tracing is not [profiling](https://github.com/produc
...
+
+Ultimately, any data structure is defined by the kind of data it contains, and relationships between data structures are defined by how links between them are recorded. Traces, transactions, and spans are no different.
+
+#### Traces
+
+Traces are defined as the collection of all transactions that share a `trace_id` value.
+
+#### Transactions
+
+Transactions share most properties (start and end time, tags, etc.) with their root spans. They also have a `transaction_name` property, used in the UI to identify the transaction. Common examples include endpoint paths for backend request transactions, task names for cron job transactions, and URLs for page-load transactions.
```
Comment (lizokm) [inline docs/concepts/key-terms/tracing/distributed-tracing.mdx:34]:
```suggestion
Transactions share most properties (start and end time, tags, and so on) with their root spans. They also have a `transaction_name` property, used in the UI to identify the transaction. Common examples include endpoint paths for backend request transactions, task names for cron job transactions, and URLs for page-load transactions.
```

File: docs/concepts/key-terms/tracing/distributed-tracing.mdx (hunk 2 above)
Comment (lizokm) [inline docs/concepts/key-terms/tracing/distributed-tracing.mdx:38]:
```suggestion

```

Comment (lizokm) [reply docs/concepts/key-terms/tracing/distributed-tracing.mdx:38]: Thanks for catching that!

File: docs/concepts/key-terms/tracing/distributed-tracing.mdx (hunk 1 above)
Comment (lizokm) [inline docs/concepts/key-terms/tracing/distributed-tracing.mdx:36]:
```suggestion
Before the transaction is sent, the `tags` and `data` properties will get merged with data from the global scope. (Global scope data is set in `Sentry.init()` or by using `Sentry.configureScope()`, `Sentry.setTag()`, `Sentry.setUser()`, or `Sentry.setExtra()`.)
```

Comment (lizokm) [reply docs/concepts/key-terms/tracing/distributed-tracing.mdx:36]: Yes!

**Pull request merged**